### DISCUSSION TITLE
What was Henry Fonda's last movie?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#720] [Galaxy Trivia] [answered]
**Bonus:** How old was he when the movie came out?

**Comment by peterbaccaro** [comment]
Henry Fonda's last movie was **On Golden Pond**.
At **76 years**, Henry Fonda was the oldest actor to win the Academy Award for Best Actor.

**Comment by thuongtruong1009** [comment, marked as answer]
Henry Fonda's last movie was "On Golden Pond," released in 1981. He was 76 years old when the movie came out.
